en/snap/platform/overview.md

```diff
@@ -383,6 +396,30 @@ The **detailed history will be displayed on Shopify admin panel's order details
+![Payment icons](./../../../asset/image/shopify-new-25-payment-icons.png ':size=400')<br>
+The selected icons will be shown on the payment button on the Shopify checkout page. It does not affect the actual payment methods on Midtrans' Snap payment page.
+![Payment icons](./../../../asset/image/shopify-new-25-payment-icons-on-checkout.png ':size=400')<br>
+If you want to configure active payment methods on the Snap payment page, you should configure it on [Midtrans Shopify Payment App Account Settings](#payment-app-account-settings).
+
+##### Shopify will not Charge Third-Party Transaction Fees for Test Order with Enabling Test Mode
+"Test order" is helpful for you to make sure that the checkout process and settings for order processing, payment, inventory, shipping, email notifications, etc are all correct. You can place a test order by [simulating a transaction through test mode](#test-transaction-with-sandbox-mode).
+
+When you use a third-party payment provider like "Midtrans" to take customer payments, shopify will add transaction fee for each transaction, the transaction fee varies depending on your [pricing plan](https://www.shopify.com/id/pricing).
+
+But if you want to do "test order" with enabling test mode, **Shopify should not charge you any payment processing fee for "test order" payments.**, all orders made within this mode are not "real", and do not require "real payment/fund", so you are **safe to ignore the payment for this order** because the payment is not real and the order should be safe to not actually fulfilled, you can check [shopify docs about test order](https://help.shopify.com/en/manual/checkout-settings/test-orders) for the detail.
+
+##### In Some Rare Cases, Midtrans Unable to Update the Payment Status to Shopify
+In some rare cases, Midtrans unable to update payment status to the Shopify, e.g: mismatch status order, status order on midtrans is settlement, but status order on shopify is pending. If it happens and affect your status order on shopify, then there are possibility:
+
+- There is an issue on the Shopify side. e.g: Shopify service is unavailable, you can check [the Shopify status page](https://www.shopifystatus.com/) for reported service outages.
+- Your store is locked by Shopify. The store is locked if your shopify account does API request and exceeds [shopify API request limit](https://shopify.dev/api/usage/rate-limits), or if there is an issue with the account, such as a detected compromise or fraud risk, posting or uploading content/product that infringes on [the copyright or trademarks](https://help.shopify.com/en/manual/your-account/copyright-and-trademark) of others, or payment failure. The impact of your store being locked are:
+	1. The customer is still able to access your store and do the checkout process, but Midtrans unable to update the payment status, this is because shopify block/reject API request from your shopify account to update the payment status
```

Comment:
> shopify

inconsistent capitalization

> from your shopify account to update the payment status

confusing explanation, what `from` refers to? is it the API request is it the blocking. Improve the explanation